### DISCUSSION TITLE
Not able to run beaglecfg

### DISCUSSION THREAD

**Discussion by GauravKumawat33** [SAtacker/beagle-config#44] [Q&A]
**Issue in running beaglecfg.**

While running the project in a local machine. After `git found`, I am getting some errors:
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads - not found
-- Found Doxygen: /usr/bin/doxygen (found version "1.8.17") found components: doxygen missing components: dot

![Screenshot from 2022-01-11 03-45-15](https://user-images.githubusercontent.com/83076014/148847971-5aa6d2f9-da4a-4e1d-910f-c7cd5472f8bc.png)
 
 
After running `make -j$(nproc)`
It's still showing `beaglecfg` and `sudo beaglecfg` is not a valid command.
 
![Screenshot from 2022-01-11 03-49-27](https://user-images.githubusercontent.com/83076014/148848197-2f72cb69-cf13-4868-89b7-53a81263a7f9.png)

I also tried running `./ cross_compile.sh` but still showing beaglecfg is not a valid command.

I am currently using Ubuntu
![Screenshot from 2022-01-11 03-58-00](https://user-images.githubusercontent.com/83076014/148849047-ae9491be-ca3c-4f82-bb2e-9349dd4665be.png)

**Comment by SAtacker** [comment]
Hello @GauravKumawat33,
The executable is not in your `$PATH` and hence it's not going to work unless you don't provide the path of the executable.

Thanks,
Shreyas

**Reply by GauravKumawat33** [reply]
can you provide the command?

and the error occurring are fine?

**Reply by SAtacker** [reply]
I encourage you to look yourself and not ask solution directly.
(Regarding the `cmake` output as well.)

**Reply by GauravKumawat33** [reply]
> I encourage you to look yourself and not ask solution directly. (Regarding the `cmake` output as well.)

Okay just give me some time to learn about it. ♡
